contract/contracts/Subscriptions.sol

```diff
@@ -202,10 +203,10 @@ contract Subscriptions {
         address user = msg.sender;
         require(user != address(0), 'user is null');
 
-        Subscription storage sub = _subscriptions[user];
+        Subscription memory sub = _subscriptions[user];
```

Comment:
Why was this changed? I think using memory here is actually more expensive because you need to copy the data from storage over to memory and we are only using it for read access.

Reply:
[240afab](https://github.com/edgeandnode/subscription-payments-contract/pull/9/commits/240afabdb803f9397b372df605cf24d6360fce2b)